bootstrap/src/lib/aws.ts

```diff
@@ -66,8 +70,12 @@ export const initHandlerHTTP = (execute: ExecuteSync) => (
   if (!isContentTypeSupported(event)) {
     return callback(null, UNSUPPORTED_MEDIA_TYPE_RESPONSE)
   }
-
-  execute(JSON.parse(event.body), (statusCode, data) => {
+  const bodyIncludingQuery = JSON.parse(event.body)
+  bodyIncludingQuery.data = {
+    ...bodyIncludingQuery.data,
+    ...toObjectWithNumbers(bodyIncludingQuery.queryStringParameters || {}),
+  }
+  execute(bodyIncludingQuery, (statusCode, data) => {
```

Comment:
This piece of code changed form, now I am ok with just calling it body.

```suggestion
  const body = JSON.parse(event.body)
  body.data = {
    ...body.data,
    ...toObjectWithNumbers(body.queryStringParameters || {}),
  }
  execute(body, (statusCode, data) => {
```